### ISSUE TITLE
examples.js: Source code syntax highlighter bug creates color problem in HTML source display for date picker combobox example

### ISSUE THREAD

**Issue by giacomo-petri** [w3c/aria-practices#2815]
In [Date Picker Combobox Example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-datepicker/) the HTML source code is not properly formatted, causing a color contrast issue for green text over light grey background, as per attachment.

<img width="863" alt="screenshot highlighting insufficient contrast ratio" src="https://github.com/w3c/aria-practices/assets/106394951/d6c1e31f-4f74-410b-9fd4-e505bc5fe795">

**Comment by giacomo-petri** [comment]
Same happens for the Example Disclosure Navigation Menu with Top-Level Links example: https://www.w3.org/WAI/ARIA/apg/patterns/disclosure/examples/disclosure-navigation-hybrid/

<img width="835" alt="screenshot highlighting insufficient contrast ratio" src="https://github.com/w3c/aria-practices/assets/106394951/84912482-452b-41de-ac26-183a6d6d744b">

**Comment by a11ydoer** [comment]
It shows as light green color because span class <span class="hljs-regexp">/div&gt;</span>was applied to this tag. 
This span class came from the directory of "conetn-assets/wai-aria-practice/shared/ css folder" and  the file is github.css
color hex code is #009926

**Comment by mcking65** [comment]
Resolved by #2939.

@giacomo-petri, thank you for raising this issue. The fix will go live in production on April 9.
